cli/src/commands/mod.rs

```diff
@@ -468,6 +468,9 @@ struct DescribeArgs {
 #[derive(clap::Args, Clone, Debug)]
 #[command(visible_aliases=&["ci"])]
 struct CommitArgs {
+    /// Interactively choose which changes to commit
```

Comment:
As I said on #2132, I think we should avoid saying "to commit" and instead say e.g. "to include in the first commit" or "to include in the parent commit".

We might also want a note that this is practically the same as `jj split`.